plugins/crd/plugin_impl_crd.go

```diff
@@ -59,11 +59,13 @@ type Plugin struct {
 	stopChNS  chan struct{}
 	stopChNSE chan struct{}
 	stopChNSC chan struct{}
-	// sharedFactory is a shared informer factory used as a cache for
-	// items in the API server. It saves each informer listing and watches the
+	// sharedFactory's are shared informer factorys used as a cache for
+	// items in the API server. They saves each informer listing and watch the
 	// same resources independently of each other, thus providing more up to
 	// date results with less 'effort'
-	sharedFactory factory.SharedInformerFactory
+	sharedFactoryNS  factory.SharedInformerFactory
```

Comment:
It is not critical, but why to continue to use SharedInformer? Since we are back to per object type Informer having three SharedInformer could be a bit more costly.

Reply:
I think now that I'm tracking object type, I can actually move back to the shared informer, so let me do that and see how much nicer this makes the PR.